### ISSUE TITLE
error on adding podio gem to project

### ISSUE THREAD

**Issue by RobertShippey** [podio/podio-rb#30]
````
/Users/robertshippey/.rvm/gems/ruby-2.2.2/gems/activesupport-4.2.1/lib/active_support/core_ext/module/deprecation.rb:21:in `deprecate': uninitialized constant ActiveSupport::Deprecation (NameError)
	from /Users/robertshippey/.rvm/gems/ruby-2.2.2/gems/activesupport-4.2.1/lib/active_support/core_ext/class/delegating_attributes.rb:26:in `<class:Class>'
	from /Users/robertshippey/.rvm/gems/ruby-2.2.2/gems/activesupport-4.2.1/lib/active_support/core_ext/class/delegating_attributes.rb:6:in `<top (required)>'
	from /Users/robertshippey/.rvm/gems/ruby-2.2.2/gems/backports-3.6.4/lib/backports/std_lib.rb:9:in `require'
	from /Users/robertshippey/.rvm/gems/ruby-2.2.2/gems/backports-3.6.4/lib/backports/std_lib.rb:9:in `require_with_backports'
	from /Users/robertshippey/.rvm/gems/ruby-2.2.2/gems/activesupport-4.2.1/lib/active_support/core_ext/class.rb:2:in `<top (required)>'
	from /Users/robertshippey/.rvm/gems/ruby-2.2.2/gems/backports-3.6.4/lib/backports/std_lib.rb:9:in `require'
	from /Users/robertshippey/.rvm/gems/ruby-2.2.2/gems/backports-3.6.4/lib/backports/std_lib.rb:9:in `require_with_backports'
	from /Users/robertshippey/.rvm/gems/ruby-2.2.2/gems/activesupport-4.2.1/lib/active_support/core_ext.rb:2:in `block in <top (required)>'
	from /Users/robertshippey/.rvm/gems/ruby-2.2.2/gems/activesupport-4.2.1/lib/active_support/core_ext.rb:1:in `each'
	from /Users/robertshippey/.rvm/gems/ruby-2.2.2/gems/activesupport-4.2.1/lib/active_support/core_ext.rb:1:in `<top (required)>'
	from /Users/robertshippey/.rvm/gems/ruby-2.2.2/gems/backports-3.6.4/lib/backports/std_lib.rb:9:in `require'
	from /Users/robertshippey/.rvm/gems/ruby-2.2.2/gems/backports-3.6.4/lib/backports/std_lib.rb:9:in `require_with_backports'
	from /Users/robertshippey/.rvm/gems/ruby-2.2.2/gems/podio-1.0.0/lib/podio.rb:2:in `<top (required)>'
	from /Users/robertshippey/.rvm/gems/ruby-2.2.2/gems/backports-3.6.4/lib/backports/std_lib.rb:9:in `require'
	from /Users/robertshippey/.rvm/gems/ruby-2.2.2/gems/backports-3.6.4/lib/backports/std_lib.rb:9:in `require_with_backports'
	from /Users/robertshippey/Desktop/status/jobs/podio.rb:2:in `<top (required)>'
...
````

I'm new to Ruby and I don't really know what this means. I'm using ruby-2.2.2 and my only other dependancies are `dashing` and `httparty`. I had a look and there was an issue already on here, #16, which looked relevant but apparently fixed so I'm unsure.

**Comment by florish** [comment]
Just to help other people bumping into this: I ran into the same issue this week on my first attempts to get this gem running. 

I suspect that, like myself, you've been trying to use the official `1.0.0` version from rubygems.org. Sadly, that version is from January 2014 and nowhere near the current master (where indeed issue #16 is fixed). The absence of an up-to-date officially released version of the gem has been reported as a separate issue in https://github.com/podio/podio-rb/issues/31.

**Comment by theflow** [comment]
This is fixed with the 2.0.0 version of the pushed gem.
